inline_scan.sh

```diff
@@ -183,9 +188,18 @@ get_and_validate_analyzer_options() {
         printf '\n\t%s\n\n' "ERROR - Manifest: ${MANIFEST_FILE} does not exist" >&2
         display_usage_analyzer >&2
         exit 1
+    elif [[ "${R_flag:-}" ]] && [[ ! -d "${PDF_DIRECTORY}" ]];then
+        printf '\n\t%s\n\n' "ERROR - Directory: ${PDF_DIRECTORY} does not exist" >&2
+        display_usage_analyzer >&2
+        exit 1
+    elif [[ "${PDF_DIRECTORY: -1}" == '/' ]]; then
+        printf '\n\t%s\n\n' "ERROR - must specify file path - ${PDF_DIRECTORY} without trailing slash" >&2
+        display_usage_analyzer >&2
+        exit 1
```

Comment:
I'd rather remove the trailing `/` and move on that report it as an error to the user.

Reply:
This was a request from the field to be like that actually, I think it's ok to leave it like this. Otherwise there will be so many other combinations we will have to do deal with. I saw one customer have multiple slashes for example .... no idea as to why !